scipy/spatial/transform/rotation.py

```diff
@@ -1431,6 +1433,20 @@ def inv(self):
             quat = quat[0]
         return self.__class__(quat, normalized=True, copy=False)
 
+    @classmethod
+    def identity(cls):
```

Comment:
identity(cls, n=None)

Similar to random

Reply:
Good idea.  Fixed.